.github/workflows/ci.yml

```
@@ -59,3 +59,64 @@ jobs:
         BUILD_FIL_NETWORK: ${{ matrix.network }}
       run: |
         cargo run --locked -- -o output/builtin-actors.car
+  coverage:
+    runs-on: ubuntu-latest
+    env:
+      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      CARGO_INCREMENTAL: 0
+      CACHE_SKIP_SAVE: ${{ matrix.push == '' || matrix.push == 'false' }}
+    steps:
+    - name: Checking out
+      uses: actions/checkout@v2
+    - name: Install Rust toolchain
+      uses: ./.github/actions/rust-cargo-run
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      with:
+        command: version
+        components: llvm-tools-preview
```

I think this should make the CI happy.

```suggestion
    - name: Install Rust toolchain
      uses: ./.github/actions/rust-cargo-run
      with:
        command: version
        components: llvm-tools-preview
        github-token: ${{ secrets.GITHUB_TOKEN }}
```